library/modules/Gui.cpp

```diff
@@ -114,14 +114,14 @@ static df::layer_object_listst *getLayerList(df::viewscreen_layer *layer, int id
 }
 */
 
-static std::string getNameChunk(virtual_identity *id, int start, int end)
+static std::string getNameChunk(virtual_identity* id, int start, int end)
```

Comment:
We don't really have a standard for `type *var` vs `type* var`, and there are merits for both, but I'd rather avoid the noisy diffs from changing it from one to the other

Reply:
Yeah this was an accidental "format selection" by the looks of it. My visual studio settings are very opinionated about the positioning of the `*` lol. Will revert for sure, along with any other style only changes that aren't driven by CI yelling at me.